DashKitUI/Views/MockPodSettingsView.swift

```diff
@@ -103,6 +105,29 @@ struct MockPodSettingsView: View {
                     ]
                 )
             }
+            Section(header: Text("System Errors").font(.headline).foregroundColor(Color.primary)) {
```

Comment:
[nit] Consider moving into `var systemErrorsHeader: some View` so it isn't all compressed into one line.